app/src/main/java/vn/sunasterisk/music_70/data/local/LocalDataSource.kt

```diff
@@ -0,0 +1,23 @@
+package vn.sunasterisk.music_70.data.local
+
+import vn.sunasterisk.music_70.data.remote.TrackDataSource
+
+class LocalDataSource : TrackDataSource.Local {
+
+    companion object {
```

Comment:
Nếu như không cần truyền tham số gì vào, ông có thể dùng object thay vì class vì object trong Kotlin đã hỗ trợ sẵn singleton rồi 
https://kotlinlang.org/docs/tutorials/kotlin-for-py/objects-and-companion-objects.html